### ISSUE TITLE
Add an option to set the maximum number of workers

### ISSUE THREAD

**Issue by norcalli** [gildas-lormeau/SingleFile#1193]
Some websites maintain server side stateful session information, and therefore loading pages in parallel can cause them to load the wrong page because the server side cursor will have a race condition. In these conditions, it would be nice to have a checkbox or alternate button that lets you "Save Pages Serially" or one by one. Also this might be nice for not overdoing CPU usage on say a laptop.

**Comment by gildas-lormeau** [comment]
You can set the number of parallel tabs by editing the options manually and setting `maxParallelWorkers` to `1`, see https://github.com/gildas-lormeau/SingleFile/wiki/Hidden-options.

**Comment by norcalli** [comment]
Ah that'll do it for me. Should I keep this open but change it to a UI ticket for adding a checkbox to expose that option?

**Comment by gildas-lormeau** [comment]
The problem is that this option is global and the UI does not present global options in a nice way. I think I would need to refactor the presentation for this.

**Comment by gildas-lormeau** [comment]
Actually, maybe this could be added in the "pending saves" view. This would make sense.
